po/zh_CN.po

```diff
@@ -132,136 +148,290 @@ msgstr "如果您需要调整该值，这可能代表存在某些缺陷。"
-
-#: daemons/controld/controld_control.c:648
+"If this is set to a positive value, lost nodes are assumed to self-fence "
+"using watchdog-based SBD within this much time. This does not require a "
+"fencing resource to be explicitly configured, though a fence_watchdog "
+"resource can be configured, to limit use to specific nodes. If this is set "
+"to 0 (the default), the cluster will never assume watchdog-based self-"
+"fencing. If this is set to a negative value, the cluster will use twice the "
+"local value of the `SBD_WATCHDOG_TIMEOUT` environment variable if that is "
+"positive, or otherwise treat this as 0. WARNING: When used, this timeout "
+"must be larger than `SBD_WATCHDOG_TIMEOUT` on all nodes that use watchdog-"
+"based SBD, and Pacemaker will refuse to start on any of those nodes where "
+"this is not true for the local value or SBD is not active. When this is set "
+"to a negative value, `SBD_WATCHDOG_TIMEOUT` must be set to the same value on "
+"all nodes that use SBD, otherwise data corruption or loss could occur."
+msgstr ""
+
+#: daemons/controld/controld_control.c:646
 msgid ""
 "How many times fencing can fail before it will no longer be immediately re-"
 "attempted on a target"
 msgstr "fence操作失败多少次会停止立即尝试"
 
-#: daemons/controld/controld_control.c:556
+
+#: daemons/fenced/pacemaker-fenced.c:1403
+msgid "Eg. node1,node2,node3"
+msgstr "例如 node1,node2,node3"
+
+#: daemons/fenced/pacemaker-fenced.c:1404
 msgid ""
-"Pacemaker is primarily event-driven, and looks ahead to know when to recheck "
-"cluster state for failure timeouts and most time-based rules. However, it "
-"will also recheck the cluster after this amount of inactivity, to evaluate "
-"rules with date specifications and serve as a fail-safe for certain types of "
-"scheduler bugs."
+"A list of machines controlled by this device (Optional unless "
+"pcmk_host_list=static-list)"
 msgstr ""
-"Pacemaker 主要是通过事件驱动的，并能预期重新检查集群状态以评估大多数基于时间"
-"的规则以及过期的错误。然而无论如何，在集群经过该时间间隔的不活动状态后，它还"
-"将重新检查集群，以评估具有日期规格的规则，并为某些类型的调度程序缺陷提供故障保护。"
+"该设备控制的机器列表(可选参数，除非 pcmk_host_list 设置为 static-list)"
 
-#: daemons/fenced/pacemaker-fenced.c:1464
-#, c-format
+#: daemons/fenced/pacemaker-fenced.c:1409
+msgid "How to determine which machines are controlled by the device."
+msgstr "如何确定设备控制哪些机器。"
+
+#: daemons/fenced/pacemaker-fenced.c:1410
 msgid ""
-" <longdesc lang=\"en\">Instance attributes available for all \"stonith\"-"
-"class resources and used by Pacemaker's fence daemon, formerly known as "
-"stonithd</longdesc>\n"
+"Allowed values: dynamic-list (query the device via the 'list' command), "
+"static-list (check the pcmk_host_list attribute), status (query the device "
+"via the 'status' command), none (assume every device can fence every machine)"
 msgstr ""
-" <longdesc lang=\"zh\">实例属性可用于所有stonith类资源，并由Pacemaker的fence"
-"守护程序使用（以前称为stonithd）</longdesc>\n"
+"允许的值：Dynamic-List（通过“列表”命令查询设备），"
+"静态列表（检查pcmk_host_list属性），状态（通过'status'命令查询设备），"
+"无（假设每个设备都可fence 每台机器 ）"
```

Comment:
Again, we should keep the original parameter values `dynamic-list`, `static-list`, `status` and `none`, the action name `list`,   untranslated and uncapitalized. Otherwise users could be confused what exactly these are referred to.